packages/css/src/details.css

```diff
@@ -32,18 +35,14 @@
     --dsc-details-border-color: var(--ds-color-neutral-border-subtle);
   }
 
-  background: var(--dsc-details-background);
-  border-block: var(--dsc-details-border);
-  box-sizing: border-box;
-
-  & > :is(summary, u-summary) {
+  & :is(summary, u-summary) {
     align-items: center;
     background: var(--dsc-details-heading-background);
     box-sizing: border-box;
     cursor: pointer;
     list-style: none;
     min-height: var(--dsc-details-size);
-    gap: var(--dsc-details-chevron-gap);
+    gap: var(--dsc-details-icon-gap);
```

Comment:
`--dsc-details-icon-spacing` for dersom vi får det i alert og skal være consistent ☺️

Reply:
Denne er eksplisitt brukt for `gap`, så den burde forbli `-gap`

Reply:
Jeg tenker fortsatt `--dsc-details-icon-spacing` fordi det ikke er viktig for sluttbrukeren hva vi bruker under panseret - det er viktigere at vi er konsekvent med navngivning og at samme custom property har samme visuelle effekt ☺️ 
Videre låser vi oss til at `summary` alltid må være `flex` eller `grid` dersom vi skal bruke `gap` property ☺️